packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -153,10 +163,16 @@ export const Tooltip = ({
   useEffect(() => {
     if (isVisible) setShow(true);
     // eslint-disable-next-line react-hooks/exhaustive-deps
+    if (!delay) {
+      if (closeOnMouseLeave) {
+        setDelaySettings({ hide: 0, show: 0 });
+      } else {
+        setDelaySettings({ hide: 500, show: 0 });
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I didn't understand why we need to do this, sorry :(

Also, there is two eslint-disable in this useEffect. Do we need two?